internal/app/table_tests.go

```diff
@@ -208,7 +208,12 @@ func shortenTestName(s string, trim bool, maxLength int) string {
 		ss := strings.Split(s, "/")
 		testName.WriteString(ss[0] + "\n")
 		for i, s := range ss[1:] {
-			testName.WriteString(" /" + s)
+			testName.WriteString(" /")
+			for len(s) > maxLength {
+				testName.WriteString(s[:maxLength] + "\n  ")
+				s = s[maxLength:]
```

Comment:
I would recommend adding an ellipsis then. Either an Unicode character (my preference) or a three successive dots


```suggestion
				s = s[maxLength:]+"…"
```